Library/brew.rb

```diff
@@ -50,6 +50,20 @@
   EOABORT
 end
 
+if OS.mac? && MacOS.version == "10.11" && MacOS::Xcode.version < "7.0"
```

Comment:
Maybe the best solution here is to revert it back to what it was before the latest push, merge that, and then take firmer action if the number of issues created because of non-system-appropriate-xcode/clt doesn't drop?